source/tests/system/nirfmxbt_driver_api_tests.cpp

```diff
@@ -104,6 +125,62 @@ TEST_F(NiRFmxBTDriverApiTests, InitializeFromNIRFSA_Close_Succeeds)
   ni::tests::system::EXPECT_SUCCESS(close_response);
 }
 
+TEST_F(NiRFmxBTDriverApiTests, AcpBasicFromExample_DataLooksReasonable)
+{
+  auto session = init_session(stub(), PXI_5663E);
+  EXPECT_SUCCESS(session, client::cfg_frequency_reference(stub(), session, "", FrequencyReferenceSource::FREQUENCY_REFERENCE_SOURCE_ONBOARD_CLOCK, 10e6));
+  EXPECT_SUCCESS(session, client::cfg_rf(stub(), session, "", 1e9, 0.0, 0.0));
+  EXPECT_SUCCESS(session, client::cfg_iq_power_edge_trigger(stub(), session, "", "0", IQPowerEdgeTriggerSlope::IQ_POWER_EDGE_TRIGGER_SLOPE_RISING_SLOPE, -20.0, 0.0, TriggerMinimumQuietTimeMode::TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO, 100e-6, IQPowerEdgeTriggerLevelType::IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_RELATIVE, Boolean::BOOLEAN_TRUE));
+  EXPECT_SUCCESS(session, client::cfg_packet_type(stub(), session, "", PacketType::PACKET_TYPE_DH1));
+  EXPECT_SUCCESS(session, client::cfg_data_rate(stub(), session, "", 1000000));
+  EXPECT_SUCCESS(session, client::cfg_payload_length(stub(), session, "", PayloadLengthMode::PAYLOAD_LENGTH_MODE_AUTO, 10));
+  EXPECT_SUCCESS(session, client::select_measurements(stub(), session, "", MeasurementTypes::MEASUREMENT_TYPES_ACP, Boolean::BOOLEAN_TRUE));
+  EXPECT_SUCCESS(session, client::acp_cfg_burst_synchronization_type(stub(), session, "", AcpBurstSynchronizationType::ACP_BURST_SYNCHRONIZATION_TYPE_PREAMBLE));
+  EXPECT_SUCCESS(session, client::acp_cfg_averaging(stub(), session, "", AcpAveragingEnabled::ACP_AVERAGING_ENABLED_FALSE, 10));
+  AcpOffsetChannelMode offsetChannelMode = AcpOffsetChannelMode::ACP_OFFSET_CHANNEL_MODE_SYMMETRIC;
+  EXPECT_SUCCESS(session, client::acp_cfg_offset_channel_mode(stub(), session, "", offsetChannelMode));
+  if (offsetChannelMode == AcpOffsetChannelMode::ACP_OFFSET_CHANNEL_MODE_SYMMETRIC)
+  {
+    EXPECT_SUCCESS(session, client::acp_cfg_number_of_offsets(stub(), session, "", 5));
+  }
+  else if (offsetChannelMode == AcpOffsetChannelMode::ACP_OFFSET_CHANNEL_MODE_INBAND)
```

Comment:
This is dead code right? I typically just pick a case and delete the other when adapting examples. You can also parameterize the test case (ping me for docs if you're interested).

Reply:
We're not exhaustively testing these drivers, not sure if there is enough benefit to test INBAND, since it appears that mode does less for this driver. I would suggest simply deleting it

Reply:
I've updated the test to just stick with the SYMMETRIC case.